src/ValueFormatters/TimeFormatter.php

```diff
@@ -17,40 +15,37 @@
  * @author H. Snater < mediawiki@snater.com >
  */
 class TimeFormatter extends ValueFormatterBase {
+
+	/**
+	 * @deprecated since 0.7, use \ValueParsers\TimeParser::CALENDAR_GREGORIAN instead.
+	 */
 	const CALENDAR_GREGORIAN = 'http://www.wikidata.org/entity/Q1985727';
+
+	/**
+	 * @deprecated since 0.7, use \ValueParsers\TimeParser::CALENDAR_JULIAN instead.
+	 */
```

Comment:
Good question. If the constant is needed in both, these two options seem equality arbitrary to me. It's also rather odd to have the formatter depend on the parser or the other way around. Also consider that it's not actually *the* time formatter or *the* time parser, rather *a* time formatter and *a* time parser.

Unless the value uses the constants, it'd not put them in there. How about creating a dedicated enum like construct?